### ISSUE TITLE
Cut-and-paste into the adjacency list

### ISSUE THREAD

**Issue by erelsgl** [yiboyang/graphrel#2]
I started with an empty graph with 4 nodes: 

```
0:[],1:[],2:[],3:[]
```

Then, I replaced the contents of the adjacency list text-box with 

```
0:[1],1:[2],2:[3],3:[0]
```

But, nothing happened to the graph - it remained edge-less.

Probably there is some problem with pasting text into the adjacency list text-box.

**Comment by erelsgl** [comment]
I found a potential source of the problem: when I pasted the text, there were leading spaces (the text I pasted was: "    0:[1],1:[2],2:[3],3:[0]"). Is there a problem with leading whitespace in the textbox?

**Comment by yiboyang** [comment]
Yep, reproduced; problem was leading/trailing whitespace; this also occurs for uploading adjacency list. Idk why [I trimmed whitespace after the regex test](https://github.com/yiboyang/graphrel/blob/bcdce681d395542483187b6dcbff7105d30c1d77/adjlist.html#L48).

**Comment by yiboyang** [comment]
Latest commit should have fixed this; unfortunately you still have to manually remove whitespace between each `parentNode:[childNodes],` pair (e.g. `0:[1],1:[0]` instead of `0:[1],   1:[0]`) for now
